src/solvers/smt2_incremental/smt_sorts.cpp

```diff
@@ -70,7 +89,6 @@ void accept(const smt_sortt &sort, const irep_idt &id, visitort &&visitor)
     return visitor.visit(static_cast<const smt_##the_id##_sortt &>(sort));
 #include "smt_sorts.def"
 #undef SORT_ID
-  UNREACHABLE;
```

Comment:
:no_entry_sign: This should not be deleted as it is indeed unreachable for well-formed sorts.

Reply:
Oh yes, sorry this was a mistake - will re-add it ASAP.